tests/follow.test.js

```diff
@@ -41,28 +40,31 @@ describe('TEST USER FOLLOW', () => {
       });
   });
 
-  it('It should return a 201 if a user is been followed successfully', done => {
+  it('should respond 201 when a user follows another user', done => {
     chai
       .request(app)
       .post('/api/v1/follow/6517a6ea-662b-4eef-ab9f-20f89bd7099c')
       .set('authorization', token1)
       .end((err, res) => {
         expect(res.status).to.equal(201);
         expect(res.body.message).to.be.a('string');
-        expect(res.body.message).to.equal('Successfully followed user');
+        expect(res.body.message).to.equal(
+          'You have successfully followed this user'
+        );
         done();
       });
   });
 
-  it('It should return a 403 if you are already following a user', done => {
+  it('It should return 403 When a user tries to follow twice', done => {
```

Comment:
You have still not refactored this @amaechi-chuks `... When a user tries to follow twice` is not a clear description.

You can do something like `It should return 403 if a user tries to follow another user he is already following`